modules/programs/ghostty.nix

```diff
@@ -78,7 +82,7 @@ in {
     installBatSyntax =
       lib.mkEnableOption "installation of Ghostty configuration syntax for bat"
       // {
-        default = true;
+        default = (cfg.package != null);
```

Comment:
Need to add a `defaultText` attribute to avoid making the documentation depend on the configuration.

Reply:
`literalExpression "(cfg.package != null)"` or `"enabled by default when package is not null`?

Reply:
Picking option 2